pkgs/development/compilers/wren/cli.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, python3
+}:
+stdenv.mkDerivation rec {
+  pname = "wren-cli";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "wren-lang";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-AUb17rV07r00SpcXAOb9PY8Ea2nxtgdZzHZdzfX5pOA=";
+  };
+
+  patches = [
+    # otherwise fails to build
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/wren-lang/wren-cli/pull/136.patch";
+      sha256 = "sha256-/c1vbc769U/WeLGZiEdnALaooOHQcpTybfbxz+KQYUM=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.